hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/TestDeleteBlocksCommandHandler.java

```diff
@@ -347,7 +379,11 @@ private static class TestSchemaHandler implements SchemaHandler {
     @Override
     public void handle(KeyValueContainerData containerData,
         DeletedBlocksTransaction tx) throws IOException {
-      // doNoting just for Test
+      if (DeleteBlocksCommandHandler.isDuplicateTransaction(containerData.getContainerID(), containerData, tx, null)) {
+        return;
+      }
```

Comment:
This doesn't quite test the whole scope of the change, but I think for this simple change it is ok given the existing structure of the test. It would be good to have this test run with a configuration that creates the containers in a tmp directory and mocked `OzoneContainer` object so we can test the whole `DeleteBlocksCommandHandler` object, but that might be out of scope for this task.